proj4/src/main/scala/geotrellis/proj4/CRS.scala

```diff
@@ -63,64 +170,56 @@ trait CRS extends Serializable {
     } else false
   }
 
-}
-
-object CRS {
-  private val crsFactory = new CRSFactory
-
   /**
-    * Creates a [[CoordinateReferenceSystem]] (CRS) from a well-known name.
-    * CRS names are of the form: "<tt>authority:code</tt>",
-    * with the components being:
-    * <ul>
-    * <li><b><tt>authority</tt></b> is a code for a namespace supported by
-    * PROJ.4.
-    * Currently supported values are
-    * <tt>EPSG</tt>,
-    * <tt>ESRI</tt>,
-    * <tt>WORLD</tt>,
-    * <tt>NA83</tt>,
-    * <tt>NAD27</tt>.
-    * If no authority is provided, the <tt>EPSG</tt> namespace is assumed.
-    * <li><b><tt>code</tt></b> is the id of a coordinate system in the authority namespace.
-    * For example, in the <tt>EPSG</tt> namespace a code is an integer value
-    * which identifies a CRS definition in the EPSG database.
-    * (Codes are read and handled as strings).
-    * </ul>
-    * An example of a valid CRS name is <tt>EPSG:3005</tt>.
-    * <p>
-    * @param name the name of a coordinate system, with optional authority prefix
-    * @return the [[CoordinateReferenceSystem]] corresponding to the given name
-    */
-  def fromName(name: String): CRS =
-    new CRS { val crs = crsFactory.createFromName(name) }
+   * Returns the WKT representation of the Coordinate Reference System
+   * @return
+   */
+  def toWKT(): String ={
+    epsgCode match {
+      case None=>"Unable to convert"
+      case Some(string) => fromNameToWKT(epsgCode.get)
+    }
+  }
 
   /**
-    * Creates a [[CoordinateReferenceSystem]]
-    * from a PROJ.4 projection parameter string.
-    * <p>
-    * An example of a valid PROJ.4 projection parameter string is:
-    * <pre>
-    * +proj=aea +lat_1=50 +lat_2=58.5 +lat_0=45 +lon_0=-126 +x_0=1000000 +y_0=0 +ellps=GRS80 +units=m
-    * </pre>
-    * @param proj4Params a PROJ.4 projection parameter string
-    * @return the specified [[CoordinateReferenceSystem]]
-    */
-  def fromString(proj4Params: String): CRS =
-    new CRS { val crs = crsFactory.createFromParameters(null, proj4Params) }
+   * Returns the WKT representation given an EPSG code in the format EPSG:[number]
+   * @param name
+   * @return
+   */
+  def fromNameToWKT(name :String): String ={
+    val code = name.split(":")(1)
+
+    val dataStream=Source.fromFile("proj4/src/main/resources/wkt/epsg.properties").getLines().toStream.dropWhile(line=>line.split("=")(0)!=code)
+
+    if(dataStream.length>0) {
+      val array = dataStream(0).split("=")
+      array(1)
+    }
+    else "Please enter a valid code"
```

Comment:
If it can't find it, should throw an exception.
new NotFoundException(s"Can't find EPSG code for $code")
creating a new exception class is a good idea